### DISCUSSION TITLE
Release tags follow the format `<name>-<version>` in a mono repo, how do I auto update?

### DISCUSSION THREAD

**Discussion by nitrocode** [renovatebot/renovate#30543] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

github

### Please tell us more about your question or problem

Id like some help configuring my service repos that reuse actions from my private mono actions repo where each release follows the pattern `<name>-<version>`

This is similar to helm chart mono repos

Please and thank you

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by RahulGautamSingh** [comment]
Try [extractVersion](https://docs.renovatebot.com/configuration-options/#extractversion) if you want to extract versions from releases fetched during lookup
If you are having problem getting versions from the package file try to [configure versioning manually](https://docs.renovatebot.com/modules/versioning/#why-you-might-need-to-manually-configure-versioning)

**Reply by nitrocode** [reply]
Hmm good call. Perhaps I can try this?

```json
{
  "packageRules": [
    {
      "matchPackageNames": ["<org>/actions/gradle-build"],
      "extractVersion": "^gradle-build-v(?<version>.*)$"
    }
  ]
}
```

This should match `gradle-build-v1.0.0` tag in `<org>/actions` and then bump up `<org>/actions/gradle-build@gradle-build-v0.9.0` to it if it finds that version in a gha workflow. Is that right?

**Reply by nitrocode** [reply, marked as answer]
This might be better

```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "packageRules": [
    {
      "matchDatasources": ["github-tags"],
      "matchDepTypes": ["action"],
      "matchPackageNames": ["<org>/actions"],
      "versionCompatibility": "^(?<compatibility>.*)-v(?<version>.*)$",
      "versioning": "semver"
    }
  ]
}
```
